honeybadgermpc/field.py

```diff
@@ -109,15 +118,7 @@ def __reduce__(self):
 
     @staticmethod
     def get(modulus):
```

Comment:
* This is redundant now. We should get rid of this. 
* I also want you to run `honeybadgermpc/ipc.py` using the `launch.sh` script. I remember we faced some issue when pickling and unpickling. As of now that's the only way to exercise that code (there are no automated tests for that).

Reply:
We should consider adding running that script in travis, but it seems to run just fine

Reply:
Yea, we were discussing running the script using a python wrapper and then making sure that things passed.